docs/sandbox/ReportApi.md

```diff
@@ -13,19 +13,30 @@ Feel free to add more reports and to add your organization to the contact info l
 
```

Comment:
If you want you can add your self or your organization to the contact info.

Reply:
Ok, I've added myself.

Reply:
Done.